### DISCUSSION TITLE
Propose Lucide icon set

### DISCUSSION THREAD

**Discussion by jclaveau** [quasarframework/quasar#17093] [Ideas / Proposals] [answered]
I just discovered the Lucide icon set which has more than 1200 icons, is open sourced and actively maintained : https://lucide.dev

I found no issue or discussion speaking about it so I wonder if the team heard about it

**Comment by yusufkandemir** [comment, marked as answer]
Not all icon libraries go into the core `@quasar/extras`. So, Lucide would fit into the `quasar-extras-svg-icons` package. But, see the notice there:
https://github.com/hawkeye64/quasar-extras-svg-icons#missing-icon-packages
> Lucide: Not true SVG. Requires a browser to create the SVGs.

It might be possible through https://github.com/lucide-icons/lucide/tree/main/packages/lucide-static, so please open an issue in https://github.com/hawkeye64/quasar-extras-svg-icons/issues so that it can be re-evaluated.
